src/main/java/kr/co/ksgk/ims/domain/invoice/controller/InvoiceController.java

```diff
@@ -0,0 +1,60 @@
+package kr.co.ksgk.ims.domain.invoice.controller;
+
+import kr.co.ksgk.ims.domain.invoice.dto.request.InvoiceUpdateRequest;
+import kr.co.ksgk.ims.domain.invoice.dto.request.UploadInvoiceInfoRequest;
+import kr.co.ksgk.ims.domain.invoice.dto.response.InvoiceInfoResponse;
+import kr.co.ksgk.ims.domain.invoice.dto.response.SimpleInvoiceInfoResponse;
+import kr.co.ksgk.ims.domain.invoice.dto.response.PagingInvoiceInfoResponse;
+import kr.co.ksgk.ims.domain.invoice.service.InvoiceService;
+import kr.co.ksgk.ims.global.common.SuccessResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/invoice")
+public class InvoiceController {
+    private final InvoiceService invoiceService;
+
+    @PostMapping
+    ResponseEntity<SuccessResponse<?>> uploadInvoice(@RequestBody UploadInvoiceInfoRequest request) {
+        SimpleInvoiceInfoResponse response = invoiceService.uploadInvoice(request);
+
+        return SuccessResponse.ok(response);
+    }
+
+    @PreAuthorize("hasRole('ADMIN')")
+    @GetMapping
+    ResponseEntity<SuccessResponse<?>> getInvoice(@RequestParam(required = false) String search, Pageable pageable) {
+        PagingInvoiceInfoResponse pagingInvoiceInfoResponse=invoiceService.getInvoiceList(search,pageable);
+
+        return SuccessResponse.ok(pagingInvoiceInfoResponse);
```

Comment:
```suggestion
        PagingInvoiceInfoResponse pagingInvoiceInfoResponse = invoiceService.getInvoiceList(search, pageable);
        return SuccessResponse.ok(pagingInvoiceInfoResponse);
```
이거 처럼 다른 모든 코드들까지 포맷팅 신경써서 한번 수정해줘
1. 불필요한 개행
2. 대입연산자(=) 주변 공백
3. 쉼표 뒤 공백 (search, pageable)